components/avatarMenu.tsx

```diff
@@ -73,20 +73,14 @@ const AvatarMenu: React.FC = () => {
           <Typography ml={1}>{loading ? "...loading" : user?.username}</Typography>
         </Box>
         <Divider />
-        {user?.role === 2 && (
```

Comment:
@ChrisShep98 we don't have to specify a hard coded value like 2 anymore because these are programmed in the database now